roles/cluster_issuer/defaults/main.yml

```diff
@@ -15,6 +15,8 @@
 cluster_issuer_name: "{{ atmosphere_ingress_cluster_issuer }}"
 cluster_issuer_type: acme
 
+cluster_issuer_acme_private_ca: false
```

Comment:
Could we technically default this to `{{ cluster_issuer_acme_server != 'https://acme-v02.api.letsencrypt.org/directory' }}` ?

Reply:
I think setting 'cluster_issuer_acme_server' does not always means a private ca is used.

Reply:
I know at least two other publicly trusted SSL certificate providers with ACME support:

- zerossl
- buypass

Reply:
TIL!